### PR TITLE
v1, does not handle error for CurrentPlayer as anonymous

### PR DESCRIPTION
**Pull request by avimoondra**
Riffing off https://github.com/avimoondra/fishbowl/pull/22

### CODE REVIEW

File: app/src/components/GameRoutes.tsx [hunk 1]
```diff
@@ -15,14 +26,58 @@ function CurrentPlayerProvider(props: {
...
+          variables: {
+            gameId: data.games[0].id,
+            clientUuid: clientUuid()
+          }
+        })
+        if (registration.data?.joinGame) {
+          await currentAuth.setJwtToken(registration.data.joinGame.jwt_token)
+        }
+        setSkipCallback(true)
+        forceUpdate()
```
Comment (avimoondra) [inline app/src/components/GameRoutes.tsx:55]: force an update, and reflow this component - which will then call `useCurrentPlayerQuery` again and we'll have a valid JWT this time (no errors)

avoids using another lazyQuery which loses some typing / generally harder to work with

Comment (wtachau) [reply app/src/components/GameRoutes.tsx:55]: ah, nice, didn't know this was possible 👍

**Comment by avimoondra** [comment]
closing in favor of https://github.com/avimoondra/fishbowl/pull/25

**Comment by avimoondra** [comment]
[[M-L] Improve Host/Join Experience](https://trello.com/c/NwI9t8lG/6-m-l-improve-host-join-experience)

**Pull request closed without merging**